pkg/apis/edgefs.rook.io/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -780,6 +785,7 @@ func (in *S3XSpec) DeepCopyInto(out *S3XSpec) {
 	*out = *in
 	in.Placement.DeepCopyInto(&out.Placement)
 	in.Resources.DeepCopyInto(&out.Resources)
+	out.ChunkCacheSize = in.ChunkCacheSize.DeepCopy()
```

Comment:
@dyusupov Could you run `make codegen` one more time.

It seems a bit weird that only `ChunkCacheSize` is added here and not also the `ResourceProfile`.

Reply:
checked - all good.